app/globals.css

```
@@ -14,7 +14,7 @@
     --foreground-hover: 0deg 0% 74%; /* for skeleton */
     --title: 0deg 0% 12%;
     --title-foreground: 0deg 0% 90%;
-    --subTitle: 0deg 0% 40%;
+    --subTitle: 0deg 0% 34%;
 
     /* The same colors */
     --border-color: 0deg 0% 40%; /*subTitle*/
```

Stylelint (17.14.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)